cirq-google/cirq_google/engine/abstract_processor.py

```diff
@@ -64,8 +64,9 @@ async def run_async(
         program_labels: Optional[Dict[str, str]] = None,
         job_description: Optional[str] = None,
         job_labels: Optional[Dict[str, str]] = None,
-        run_name: str = "",
-        device_config_name: str = "",
+        *,
```

Comment:
do we need the `*` here and elsewhere? what is the purpose?

Reply:
Mainly for readability and convenience. This preserves parameter ordering and keeps the number of required changes low.

Reply:
On a second look, it is a bit strange to have optional positional arguments followed by mandatory kw arguments `run_name`, `device_config_name`.  I tried to move the added `*,` just before the first optional arguments and the tests pass.

Can we do so and also move the `run_name`, `device_name` as the first arguments after `*,`?
The order of kw-only arguments does not matter, but for code readability it is nicer to have
mandatory arguments followed by optional.

Reply:
Good point! Moved.